src/poliastro/twobody/orbit.py

```diff
@@ -294,12 +297,17 @@ def sample(self, values=None, function=propagate):
 
         elif isinstance(values, int):
             if self.ecc < 1:
-                nu_values = np.linspace(0, 2 * np.pi, values) * u.rad
+                # first sample eccenric anomaly, then transform into true anomaly
```

Comment:
s/eccenric/eccentric/g

Also, what about adding a reference to the report in the comment? "Why sample uniformly on the eccentric anomaly to minimize error in the apocenter, see ..."